packages/nx/schemas/nx-schema.json

```diff
@@ -157,11 +157,16 @@
                     "$ref": "#/definitions/NxReleaseVersionConfiguration"
                   },
                   {
-                    "anyOf": [
+                    "allOf": [
                       {
                         "not": {
                           "required": ["git"]
                         }
+                      },
+                      {
+                        "not": {
+                          "required": ["preVersionCommand"]
+                        }
```

Comment:
I don't think this should be necessary (it shouldn't be for `git` either). Props are not required by default in json schema.

Reply:
This is inverting the logic - those properties are defined in the NxReleaseVersionConfiguration above but are explicitly not allowed in this "version" prop